crates/configuration/src/relaychain.rs

```diff
@@ -445,6 +452,38 @@ impl RelaychainConfigBuilder<WithChain> {
             ),
         }
     }
+
+    pub fn with_group(
```

Comment:
```suggestion
    pub fn with_node_group(
```
Also, this methods needs docs `///`